test/SparkConduit.t.sol

```diff
@@ -1250,13 +1253,309 @@ contract SparkConduitGettersTests is SparkConduitTestBase {
 
         IInterestRateDataSource.InterestData memory data = conduit.getInterestData(address(token));
 
-        assertApproxEqRel(data.baseRate,    400 * RBPS, WBPS);
-        assertApproxEqRel(data.subsidyRate, 350 * RBPS, WBPS);
+        // TODO: Investigate reducing diff
+        assertApproxEqAbs(data.baseRate,    400 * RBPS, 1e9);
+        assertApproxEqAbs(data.subsidyRate, 350 * RBPS, 1e9);
 
         assertEq(data.currentDebt, 100 ether);
         assertEq(data.targetDebt,  60 ether);
     }
 
+    function testFuzz_getInterestData(
+        uint256 subsidySpread,
+        uint256 dsrAnnualRate,
+        uint256 depositAmount,
+        uint256 requestAmount
+    )
+        external
+    {
+        subsidySpread = _bound(subsidySpread, 0, 500_00 * RBPS);
+        dsrAnnualRate = _bound(dsrAnnualRate, 0, 500_00 * RBPS);
+        depositAmount = _bound(depositAmount, 0, 1e32);
+        requestAmount = _bound(requestAmount, 0, depositAmount);
+
+        conduit.setSubsidySpread(subsidySpread);
+        pot.setDSR(dsrAnnualRate / SECONDS_PER_YEAR + RAY);
+
+        token.mint(buffer, depositAmount);
+        conduit.deposit(ILK, address(token), depositAmount);
+        deal(address(token), address(atoken), 0);
+        conduit.requestFunds(ILK, address(token), requestAmount);
+
+        IInterestRateDataSource.InterestData memory data = conduit.getInterestData(address(token));
+
+        // TODO: Investigate reducing diff
+        assertApproxEqAbs(data.baseRate,    dsrAnnualRate + subsidySpread, 1e9);
+        assertApproxEqAbs(data.subsidyRate, dsrAnnualRate,                 1e9);
+
+        // TODO: Figure out why this isn't working
```

Comment:
Why is this TODO here? Seems like it's working?

Reply:
Yep you're right, removed comment